elytron-common/pom.xml

```diff
@@ -28,4 +28,165 @@
     <artifactId>wildfly-elytron-integration-common</artifactId>
 
     <name>WildFly: Elytron Subsystem - Common Components</name>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.wildfly.core</groupId>
+                <artifactId>wildfly-core-testbom</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.authentication</groupId>
+            <artifactId>jakarta.authentication-api</artifactId>
```

Comment:
Just to check, where is this needed?

Reply:
It's only used by [AbstractElytronSubsystemBaseTest](https://github.com/cam-rod/wildfly-core/blob/194a812265cf11165c061572121f2c9fc631d392/elytron-common/src/test/java/org/wildfly/extension/elytron/common/AbstractElytronSubsystemBaseTest.java#L19) for the `DEFAULT_SECURITY_FACTORY_PROPERTY`. I'll relabel it as a test dependency.